distributions/otelcol-ebpf-profiler/manifest.yaml

```diff
@@ -2,33 +2,33 @@ dist:
   module: github.com/open-telemetry/opentelemetry-collector-releases/ebpf-profiler
   name: otelcol-ebpf-profiler
   description: OpenTelemetry Collector for eBPF Profiling
-  version: 0.123.1
+  version: 0.125.0
   output_path: ./_build
 
 exporters:
-  - gomod: go.opentelemetry.io/collector/exporter/debugexporter v0.123.0
-  - gomod: go.opentelemetry.io/collector/exporter/nopexporter v0.123.0
-  - gomod: go.opentelemetry.io/collector/exporter/otlpexporter v0.123.0
-  - gomod: go.opentelemetry.io/collector/exporter/otlphttpexporter v0.123.0
-  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/fileexporter v0.123.0
+  - gomod: go.opentelemetry.io/collector/exporter/debugexporter v0.125.0
+  - gomod: go.opentelemetry.io/collector/exporter/nopexporter v0.125.0
+  - gomod: go.opentelemetry.io/collector/exporter/otlpexporter v0.125.0
+  - gomod: go.opentelemetry.io/collector/exporter/otlphttpexporter v0.125.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/exporter/fileexporter v0.125.0
 
 processors:
-  - gomod: go.opentelemetry.io/collector/processor/batchprocessor v0.123.0
-  - gomod: go.opentelemetry.io/collector/processor/memorylimiterprocessor v0.123.0
-  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/k8sattributesprocessor v0.123.0
-  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourcedetectionprocessor v0.123.0
-  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourceprocessor v0.123.0
+  - gomod: go.opentelemetry.io/collector/processor/batchprocessor v0.125.0
+  - gomod: go.opentelemetry.io/collector/processor/memorylimiterprocessor v0.125.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/k8sattributesprocessor v0.125.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourcedetectionprocessor v0.125.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourceprocessor v0.125.0
 
 receivers:
   - gomod: go.opentelemetry.io/ebpf-profiler v0.0.0-20250408174414-ead430a15d49
```

Comment:
@dmathieu what version should be used for the new release for this?

Reply:
Another question is does this version work with other 0.125.0 components ?

Reply:
ebpf-profiler doesn't cut releases yet. So we need to keep relying on commits.
I'm not sure we should upgrade them alongside the release. But we could maybe setup a weekly upgrade?

Reply:
> But we could maybe setup a weekly upgrade?

This version won't be auto-upgraded by the workflow / script, so a weekly upgrade makes sense to me.

So for this release it is fine to use the current commit version + v0.125.0 of other otel packages?

Reply:
Yes.